frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -347,7 +363,7 @@ object Interpreter {
             else doJsRun(project, config)(state)
 
           case Platform.Jvm(_) =>
-            val msg = "Expected Scala Native or Scala.js project to run `link`."
+            val msg = "Only Scala.js and Scala Native projects can be linked"
```

Comment:
I prefer to use well-structured and easy to parse error messages that start with "Expected" or "Missing". What about "Link failed. ${project.name} is not a Scala Native or Scala.js project."?

Reply:
That's better. How about the following:

> Cannot link JVM project ${project.name}. `link` is only available for Scala Native and Scala.js projects.

Reply:
Perfect!